chromium_presubmit_config.json5

```diff
@@ -58,6 +58,7 @@
   "default_files_to_skip": [
     "\\.storybook/",
     "components/brave_wallet/resources/solana_web3_script\\.js",
+    "components/brave_wallet_ui/mocks/mock-dapps-list.ts",
```

Comment:
why we need to skip this?

Reply:
Removed `mock-dapps-list.ts` from excluded files